docs/docs/display_conditions.md

```diff
@@ -0,0 +1,499 @@
+The `displayConditions` key controls how, when, and for how long any single module is displayed/hidden. All options can be combined with one-another for maximum control. Each module requires its own set of `displayConditions`. By default all modules will leverage `showOnInit`. Upon configuring another options this init based display is automatically disabled unless explicitly defined again in the module config.
+
+
+``` javascript
+var module = pathfora.Message({
+  displayConditions: {
+    // display conditions here
+  }
+});
+```
+
+## showOnInit
+Determines if the rendered module is displayed as soon as it is initialized or waits for another event.
+
+| Value | Type | Behavior |
+|---|---|---|
+| true | bool | `default` module will be displayed as soon as it has been loaded |
+| false | bool | module will be added to DOM but not displayed until another trigger instructs it to |  
+
+``` javascript
+// example: loads immediately after initializing the module
+
+displayConditions: {
+  showOnInit: true
+}
+```
+
+## showDelay
+Adds a delay, in seconds, that must be completed before module is loaded.
+
+| Value | Type | Behavior |
+|---|---|---|
+| 0 | int | `default` using 0 disables the showDelay property and will show immediately |
+| 0 - ∞ | int | module will be displayed after waiting x seconds after initialization |  
+
+
+``` javascript
+// example: loads after a 10 second delay
+
+displayConditions: {
+  showDelay: 10
+}
+```
+
+## hideAfter
+Adds a countdown, in seconds, that must hides module on expiration.
+
+| Value | Type | Behavior |
+|---|---|---|
+| 0 | int | `default` using 0 disables the showDelay property and will not be hidden |
+| 0 - ∞ | int | module will hidden from screen after x seconds have passed |  
+
+``` javascript
+// example: hide module after 10 seconds
+
+displayConditions: {
+  hideAfter: 10
+}
+```
+
+## displayWhenElementVisible
+Triggers the module when a specific DOM element enters the viewport.
+
+| Value | Type | Behavior |
+|---|---|---|
+| element id | string | waits for an element with specific id to enter the viewport |
```

Comment:
It was at one point? Though looking at the source code it looks like it got changed to any selector, which is good, and we would have done at some point anyway! Will update the docs then.